monai/apps/detection/transforms/array.py

```diff
@@ -535,7 +535,8 @@ class RotateBox90(Rotate90):
     def __init__(self, k: int = 1, spatial_axes: Tuple[int, int] = (0, 1)) -> None:
         super().__init__(k, spatial_axes)
 
-    def __call__(self, boxes: NdarrayOrTensor, spatial_size: Union[Sequence[int], int]) -> NdarrayOrTensor:  # type: ignore
+    # type: ignore
+    def __call__(self, boxes: NdarrayOrTensor, spatial_size: Union[Sequence[int], int]) -> NdarrayOrTensor:
         """
```

Comment:
This is auto format?
Please change it to fix mypy check:
```py
def XXX(  # ignore ...
    XXX
)
```

Thanks.

Reply:
I think so! I revert them to be as they were!